src/test/java/com/xceptance/neodymium/util/LighthouseUtilsTest.java

```diff
@@ -0,0 +1,212 @@
+package com.xceptance.neodymium.util;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+import org.junit.Assert;
+
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.common.browser.Browser;
+import com.xceptance.neodymium.junit5.NeodymiumTest;
+
+@Browser("Chrome_headless")
+public class LighthouseUtilsTest
+{
+    @NeodymiumTest
+    public void testLighthouseUtilsHappyPath() throws Exception
+    {
+        Neodymium.configuration().setProperty("neodymium.lighthouse.performance", "0.5");
+        Neodymium.configuration().setProperty("neodymium.lighthouse.accessiblity", "0.5");
+        Neodymium.configuration().setProperty("neodymium.lighthouse.bestPractices", "0.5");
+        Neodymium.configuration().setProperty("neodymium.lighthouse.seo", "0.5");
+        
+        ProcessBuilder builder = new ProcessBuilder();
+        
+        if (System.getProperty("os.name").toLowerCase().contains("win")) 
+        {
+            Neodymium.configuration().setProperty("neodymium.lighthouse.binaryPath", "echo {\"categories\": {\"performance\": {\"score\": 0.5}, \"accessibility\": {\"score\": 0.5}, \"best-practices\": {\"score\": 0.5}, \"seo\": {\"score\": 0.5}}} > target/lighthouseUtilsReport.report.json | echo makeCommentWork #");
+            builder = new ProcessBuilder("cmd.exe", "/c", "fabricatedHtml > target/lighthouseUtilsReport.report.html");
+        }
+        else if (System.getProperty("os.name").toLowerCase().contains("linux") || System.getProperty("os.name").toLowerCase().contains("mac"))
+        {
+            Neodymium.configuration().setProperty("neodymium.lighthouse.binaryPath", "echo {\"categories\": {\"performance\": {\"score\": 0.5}, \"accessibility\": {\"score\": 0.5}, \"best-practices\": {\"score\": 0.5}, \"seo\": {\"score\": 0.5}}} > target/lighthouseUtilsReport.report.json");
+            builder = new ProcessBuilder("fabricatedHtml > target/lighthouseUtilsReport.report.html");
```

Comment:
this needs to be changed to `new ProcessBuilder("sh", "-c", "echo fabricatedHtml > target/lighthouseUtilsReport.report.html");`